1-js/02-first-steps/04-variables/article.md

```diff
@@ -42,7 +43,8 @@ alert(message); // محتوای متغیر را نشان می‌دهد
 برای اختصار می‌توانیم تعریف متغیر و مقداردهی به آن را در یک خط قرار دهیم :
 
 ```js run
-let message = 'سلام!'; // متغیر را تعریف کن و مقدار دهی کن
+ //متغیر را تعریف کن و مقدار دهی کن
+let message = 'سلام!';
```

Comment:
This will cause merge conflicts in the future upstream PRs.

Maybe it's better to translate the string to "Hello!", so the comment wouldn't be combined with it.